.github/tests/spire-oidc-insecure/values.yaml

```diff
@@ -1,3 +1,23 @@
+spire-server:
+  ingress:
+    enabled: true
+    className: "nginx"
+    annotations:
+      nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+      nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+      nginx.ingress.kubernetes.io/ssl-redirect: "true"
+      nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+    hosts:
+    - host: ingress-nginx-controller.spire-oidc-insecure
```

Comment:
Should this reference spire server since that is the target?

Reply:
The ingress controller that the test is running in is in the spire-oidc-insecure namespace. so that is a valid hostname to use for testing against.